book.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
Getters and Setters for `rentals` should be present, Kindly add the `rentals` to attr_accessor. 👍